projects/step-frontend/src/lib/modules/execution/components/schedule-overview/schedule-overview.component.html

```diff
@@ -0,0 +1,4 @@
+<div>
+  <step-icon name="clock"></step-icon>
```

Comment:
This is totally correct for the current status, but as we might change the scheduler icon at some point we need to do it a bit more dynamically. Check out <entity-icon> (step-frontend/projects/step-core/src/lib/modules/entity/components/entity-icon/entity-icon.component.ts) 
We need to provide the entityName and also the entity object